conf/livy-defaults.conf.template

```diff
@@ -38,3 +38,6 @@
 # or an hdfs:// directory. Be careful setting this option, as it may allow users to access
 # any file the Livy process can access, such as Livy's SSL certificate.
 ## livy.files.dir = /var/run/livy/uploaded-files
+
+# Location to sparkr package file(sparkr.zip) with symbolic link
```

Comment:
Doesn't the archive always need to be extracted to `sparkr` on the YARN side? You should't tell the user to do that, much else allow the user to override that. The code in Livy reading this configuration should make sure the fragment (i.e. anything after the '#') is ignored.